tests/store/events.spec.ts

```diff
@@ -0,0 +1,93 @@
+import createStore from '~/store'
+
+describe('module events', () => {
+  let store: any
+
+  const unconferenceCampInAsia = {
+    title: 'Fake event A',
+    type: ['Unconference', 'Camp'],
+    location: 'Asia Pacific'
+  }
+  const hackathonInEurope = {
+    title: 'Fake event B',
+    type: ['Unconference', 'Hackathon'],
+    location: 'Europe'
+  }
+  const campInAfrica = {
+    title: 'Fake event C',
+    type: ['Camp'],
+    location: 'Africa'
+  }
+  const hackathonInAmericas = {
+    title: 'Fake event D',
+    type: ['Camp', 'Hackathon'],
+    location: 'Americas'
+  }
+  const unconferenceOnline = {
+    title: 'Fake event E',
+    type: ['Unconference'],
+    location: 'Online'
+  }
+
+  const futureEvents = [unconferenceCampInAsia, hackathonInEurope, campInAfrica]
+  const pastEvents = [hackathonInAmericas, unconferenceOnline]
+
+  beforeEach(() => {
+    store = createStore()
+    store.commit('setEvents', {
+      events: 'upcomingCommunityEvents',
+      eventsSet: futureEvents
+    })
+    store.commit('setEvents', {
+      events: 'pastCommunityEvents',
+      eventsSet: pastEvents
+    })
+  })
+
+  describe('filteredEvents', () => {
+    it('get future events by default', () => {
```

Comment:
Non-blocker. Take a look if you have time to [this issue](https://github.com/Qiskit/qiskit.org/issues/522) I created about the tests descriptions and let me know what you think